scripts/publish.sh

```diff
@@ -12,6 +12,7 @@ VER=`./scripts/getversion.js`
 cp ./wikipediapreview.php $1/trunk/
 cp ./readme.txt $1/trunk/
 cp ./assets/js/* $1/trunk/assets/js/
+cp ./build $1/build/
```

Comment:
I'm still not very sure about the folder structure yet,

the `start` `build` script **remove entire `folder` and generate the code in `build` folder**, therefore we can't just use the `assets` folder

we place the `wikipedia-preview` script and style in `assets`

therefore in `wikipediapreview.php`  has two dir `$assets_dir` `$build_dir`, this brings confuse: when to use `assets` when to use `build`.

any idea?